Portfolio - First Mobile design

HFG43 opened the pull request:
Hi! Many thanks for the previous CR.
I have made requested and suggested changes to my project.
Thanks for your feedback!

Hi there, I'm sending this PR for you to review my code. Thanks in advance for your feedback!

For this project, I have done the following:
* Follow the use of html tags as Best Practice
* Used Linters
* Follow FIGMA design of Template 1, to prepare my portfolio showcase
* Use Flex-Box to position elements in Navigation Bar and Icons
* Use basis of responsive design using percentages instead of pixels

HFG43 committed:
Add .gitignore

HFG43 committed:
Nav Bar in place & Presentation info

NavBar with logo, finished
Include info and icons

HFG43 committed:
Introduction Section text & icons

Text and Icons in place
Background remaining

HFG43 committed:
Place Backgorund

Place Background
Correct padding/margin in presentation section, to aloud BG to set in place
Review html tags

HFG43 committed:
Add README

Add Readme template

HFG43 committed:
Linters Correction in CSS Style

KanzaTahreem commented on index.html:
```diff
@@ -0,0 +1,38 @@
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li><img class="my-logo" src="Images_Background/Logo HG.png" alt="Personal Logo"></li>
```
- [ ] Good job for adding a personalized logo 🥇 However, please make sure to follow the design guidelines to get a design close to Figma template as much as possible according to the project requirements ✨ Kindly replace it with the text present in the design guidelines 😃
- [ ] Also, please make sure that the logo is a link 🔗, which means that it has an `a` tag with `href` attribute. This is because of convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information ℹ

KanzaTahreem commented on index.html:
```diff
@@ -0,0 +1,38 @@
+      <p class="introduction-paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="lets-connect">LET'S CONNECT</h2>
+      <ul class="social-icons">
+        <li><a href="#"><img src="Images_Background/Twitter.svg" alt="Twitter Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/M.svg" alt="M Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/GitHub.svg" alt="GitHub Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/Fingers.svg" alt="Fingers Icon"></a></li>
```
- [OPTIONAL] I think it would be better if you add the links of your actual social profiles ✨ This is a great way to build a network. What do you think about it? 💭

KanzaTahreem commented on style.css:
```diff
@@ -0,0 +1,78 @@
+
+/* INTRODUCTION SECTION */
+
+#introduction-section {
+  background: url("Images_Background/header-shapes\ mobile.png") no-repeat;
+  background-size: cover;
+}
```
- [ ] Please make sure that the content of the section is aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive.💪 [Check out this link](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) for more information ℹ
- [ ] Kindly make sure that the background image is not cropped. Right now, I can see that the background image is not according to what is given in the design guidelines. See the images below 👇 _Hint: This is related to height_
- [ ] Please consider adding a `border-bottom-left-radius` for the first section. Right now, it is missing. It may not be visible now, but when you will start working on the next milestone of the portfolio project, it will be visible. See the image below for more clarity 💡
- [OPTIONAL] I think it would be much better to use `.svg` instead of `.png` 💪 While `pngs` are capable of handling very high resolutions, they're not infinitely expandable. On the other hand, `svg` files can expand to any size without losing their resolution ℹ

Your design | Design guidelines
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/234743022-87e0f981-9824-442f-888f-4c3a8fccb4b0.png) | ![image](https://user-images.githubusercontent.com/106119105/234743272-21aa6b2e-9098-428f-baee-151ab844d362.png)

KanzaTahreem commented on README.md:
```diff
@@ -1 +1,174 @@
+
+- GitHub: [@githubhandle](https://github.com/HFG43)
+- Twitter: [@twitterhandle](https://twitter.com/HFG_43)
+- LinkedIn: [LinkedIn](linkedin.com/in/hernán-güemes-a440591b)
```
- [ ] Very good job customizing this section 🥇 I noticed that the LinkedIn link is not working. I'm getting the following error when I'm clicking on it 👇 Please fox this ⚒

![image](https://user-images.githubusercontent.com/106119105/234744294-7135186a-fa7e-4c6b-922e-35d6fde1995e.png)

KanzaTahreem commented on README.md:
```diff
@@ -1 +1,174 @@
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+**[dinamic_style_media_querys]**
+**[RWD_Responsive_Web_Design]**
+**[contact_form]**
+**[accesibility]**
+**[add_portfolio_snapshots]**
+**[deployment]**
+
```
- [OPTIONAL] Good job adding **_future features_**🥇 I think it would be better if you add them like the way you added **_key features_** What do you think? 🤔

Future features | Key features
---| ---|
![image](https://user-images.githubusercontent.com/106119105/234744655-385604f3-81ad-460f-a4d9-0eafe4edccd4.png) | ![image](https://user-images.githubusercontent.com/106119105/234744703-1a457aa3-8d6f-4a66-8ba7-a3d3b0ff64a5.png)

KanzaTahreem reviewed (changes requested):
# Changes Required 🔁

Hi @HFG43 👋

Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! 

**_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

![so_close](https://user-images.githubusercontent.com/99715640/211022680-440a1bb1-4281-4acb-8dbc-3442fe509063.gif)

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done following the given design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HFG43 committed:
My Logo correction

Remove personal Logo, to follow FIGMA design

HFG43 committed:
Correct links

Correct links in Media Icons & in README file

HFG43 committed:
Review changes in BackGround

HFG43 committed:
Change BG Image

Change file to SVG instead of png

HFG43 committed:
Container Flex-Box & README

Correct README file with all suggestions & included Flex-Box for presentation container

HFG43 committed:
Correct BG

HFG43 committed:
BG in place

V-Blaze commented on style.css:
```diff
@@ -0,0 +1,91 @@
+
+/* NAVIGATION BAR */
+
+.navbar {
+  margin: 46px 6.5% 0 6.5%;
+}
```
-  [OPTIONAL] Great job implementing the changes requested by the previous reviewer here. :+1: :+1: I noticed that the margin-top is slightly higher than the one in the given design. I'll suggest you reduce the margin to match the given Figma design. changing it to this works fine `.navbar {
  margin: 8px 6.5% 0 6.5%;
}`

- SCREENSHOT REFERENCE OF THE EXTRA MAGIN-TOP
![Selection_015](https://user-images.githubusercontent.com/107632780/234949566-e54ed147-d8aa-4c1f-9b5f-ed7ea1c89e7b.png)

HFG43 replied:
Hi Valentine! Many thanks for your feedback!

V-Blaze reviewed (approved):
Hi @HFG43  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![giphy](https://user-images.githubusercontent.com/107632780/229644741-7dea3027-bf3f-4416-855b-46095e3bf6d5.gif)

## Highlights

- Portfolio matches the Figma design💯
-  Readme is professional👍
-  Great communication with the reviewer🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HFG43 committed:
NavBar top Margin

Reduce TOP Margin
